.github/CODEOWNERS

```diff
@@ -324,8 +324,8 @@ reports/distributions/k8s.yaml     @open-telemetry/collector-contrib-approvers
 
 ## UNMAINTAINED components
 
-exporter/kineticaexporter/                                       @open-telemetry/collector-contrib-approvers
-exporter/opensearchexporter/                                     @open-telemetry/collector-contrib-approvers
-extension/observer/ecstaskobserver/                              @open-telemetry/collector-contrib-approvers
-extension/storage/filestorage/                                   @open-telemetry/collector-contrib-approvers
-receiver/jmxreceiver/                                            @open-telemetry/collector-contrib-approvers
+exporter/kineticaexporter/                                       @open-telemetry/collector-contrib-approvers 
```

Comment:
Why is extra whitespace being added here? I assume it's to fix CI?

Reply:
Yes, without the whitespace CI failed: https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/12832998175/job/35787022502

(it's really hard to tell unless you select and highlight those diffs)

Reply:
I'm fine with merging this now to fix CI, I realize it's time sensitive, but it seems like the real solution here is to fix `githubgen` to not add whitespace after these lines. I assume removing the trailing whitespace before `\n` should fix it?
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3febf16a466608080535419ed25297827e0e141b/cmd/githubgen/codeowners.go#L142

Reply:
👍 created https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/37292 to follow up, will merge this PR to fix mainline for now